sim-administration/postgres/init.sql

```diff
@@ -22,6 +29,9 @@ create table sim_entries (id bigserial primary key,
                           pin2 varchar(4),
                           puk1 varchar(8),
                           puk2 varchar(8),
+                          tsHlrState timestamp default now(),
+                          tsSmdpPlusState timestamp default now(),
+                          tsProvisionState timestamp default now(),
```

Comment:
So, these are not the timestamps for every state transition, but instead the timestamp when the last state transition happened for that state.  If that's sufficient for the use-case,  then good! 👍